src/commands/synthetics/api.ts

```diff
@@ -27,7 +27,8 @@ import {ciTriggerApp, getDatadogHost, retry} from './utils/public'
 const MAX_RETRIES = 3
 const DELAY_BETWEEN_RETRIES = 500 // In ms
 const LARGE_DELAY_BETWEEN_RETRIES = 1000 // In ms
-const DELAY_FOR_TOO_MANY_REQUESTS = 5000 // In ms (5s). Could be changed to the header returned for 429s later on
+// SYNTH-13709: Use the `Retry-After` header.
+const DELAY_BETWEEN_429_RETRIES = 5000 // In ms (5s). Could be changed to the header returned for 429s later on
```

Comment:
**nit**
- Either add comments everywhere to say `500ms, 1s, 5s` or only keep `In ms` everywhere, but stay consistent.
- And "Could be changed to the header returned for 429s later on" is redundant: the comment above with `SYNTH-13709` says it all. If you want you can update it to "Use the `Retry-After` header returned for 429s"
```suggestion
const DELAY_BETWEEN_429_RETRIES = 5000 // In ms
```

Reply:
Interesting i was sure i removed that 🤔